### PR TITLE
Update/leaflet version

### PR DESCRIPTION
**Pull request by markstuart**
This brings Mappy up to date with Leaflet v1.0.3

This was needed to fix issues with rendering and long-running script popups in a client project, and it makes sense in general to bump our main dependency frequently to keep pace with development.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,9 +56,9 @@
     "foreach": "2.0.5",
     "handlebars": "2.0.0",
     "insert-css": "0.2.0",
-    "leaflet": "^0.7.5",
-    "leaflet-plugins": "^1.3.10",
-    "leaflet.markercluster": "^0.4.0",
+    "leaflet": "1.0.3",
+    "leaflet.gridlayer.googlemutant": "0.4.3",
```
Comment (digitalsadhu) [inline package.json]: wtf is this?

File: wrappers/tilelayer.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ module.exports = function (name, config) {
   var tileLayer = null
 
   if(config.type === 'google') {
-    tileLayer = new L.Google((config.mapType ? config.mapType : 'ROADMAP'))
+    tileLayer = L.gridLayer.googleMutant({ type: config.mapType || 'roadmap' })
```
Comment (digitalsadhu) [inline wrappers/tilelayer.js]: double checking 'roadmap' changed from 'ROADMAP' was intended?

**Review by digitalsadhu — APPROVED** [review]
NOTE: Not tested
LGTM (with conditions) See final comment

File: package.json (hunk 1 above)
Comment (markstuart) [inline package.json]: The only thing we were using from leaflet-plugins was the Google tile layer plugin. That's non-existent in the latest leaflet-plugins, googlemutant is the new black apparently

File: wrappers/tilelayer.js (hunk 2 above)
Comment (markstuart) [inline wrappers/tilelayer.js]: Modelled off the googlemutant readme example

**Pull request merged**